clients/githubrepo/roundtripper/transport.go

```diff
@@ -64,5 +64,11 @@ func (gt *githubTransport) RoundTrip(r *http.Request) (*http.Response, error) {
 	if err == nil {
 		stats.Record(ctx, githubstats.RemainingTokens.M(int64(remaining)))
 	}
+
+	retryAfter, err := strconv.Atoi(resp.Header.Get("Retry-After"))
+	if err == nil {
+		stats.Record(r.Context(), githubstats.RetryAfter.M(int64(retryAfter)))
+	}
+
```

Comment:
Shouldn't this be in the `rate_limit.go` file where we handle `Retry-After`? Otherwise we'll always record `0` since `Retry-After` would be handled already.

Reply:
Good catch. I haven't observed any `retry-after` headers yet, so had no indication this was working/broken.